test/integration/infra/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.7"
+# SPDX-FileCopyrightText: 2022-present Open Networking Foundation
+# SPDX-License-Identifier: Apache-2.0
+
+services:
+  pfcpiface:
+    build:
+      context: ../../../
+      dockerfile: Dockerfile
+      target: pfcpiface
+    hostname: pfcpiface
+    container_name: pfcpiface
+    tty: true
+    stdin_open: true
+    volumes:
+      - ../config/upf.json:/config.json
+    entrypoint: "/bin/pfcpiface"
+    command: [ "-config", "/config.json" ]
+    working_dir: /bin/
+    ports:
+      - "8805:8805/udp"
+    expose:
+      - 8805
+    depends_on:
+      - mock-up4
+  mock-up4:
+    image: opennetworking/p4mn
```

Comment:
To facilitate reproducibility, should we provide the image sha?

Reply:
Also, I recommend using the the `mn-stratum` image with `stratum_bmv` (instead of `simple_switch_grpc` in `p4mn`), since we directly maintain that implementation of P4Runtime (it's easier to fix bugs).

Reply:
I believe we still need to use `simple_switch_grpc` for DDN (digest). I tried to use `stratum_bmv2` on UP4 STC and the digest test still fails, even if this https://github.com/stratum/stratum/issues/528 was solved.

Reply:
ahh I've already changed, but considering DDNs I'm going to revert it.